### PR TITLE
Added Wifite2, SecLists, Image and some minor changes.

### PR DESCRIPTION
**Pull request by krishna8421**
**Content Update: **  Yes

**Related Content: ** Tools

**Changes**

- Added Wifite2 in Wireless hacking Tool.
- Added SecLists in Wordlist
- Changed the Image
- Added a site link in Additional resources
- Some Minor edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-<p align="center">
-    <img src="https://i.imgur.com/VeEYEkT.png" alt="Hacker roadmap" /><br>
-</p>
+<div style="text-align:center">
+  <img src="img/hacker-roadmap.jpeg" alt="Hacker Roadmap" />
```
Comment (sundowndev) [inline README.md:2]: Please keep the old logo as PNG format

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 **Related Content :** Guide|Tools
 
-*Explain your changes here. Please read CONTRIBUTING.md before creating pull requests.*
+*Explain your changes here. Please read [CONTRIBUTE.md](CONTRIBUTE.md) before creating pull requests.*
```
Comment (sundowndev) [inline PULL_REQUEST_TEMPLATE.md:5]: What's the matter of that edit? GitHub already recognize "CONTRIBUTING.md"

File: README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+</div>
+<br>
 
-This repository is an overview of what you need to learn penetration testing and a collection of hacking tools, resources and references to practice ethical hacking. Most of the tools are UNIX compatible, free and open source.
+This repository is an overview of what you need to learn in penetration testing and a collection of hacking tools, resources and references to practice ethical hacking. Most of the tools are UNIX compatible, free and open source.
```
Comment (sundowndev) [inline README.md:6]: This sentence is already correct : 

> This repository is an overview of what you need ; to learn penetration testing

File: README.md [hunk 4]
```diff
@@ -296,6 +300,7 @@ Frameworks are packs of pen testing tools with custom shell navigation and docum
 - [CWE](http://cwe.mitre.org/)
 - [NVD](http://web.nvd.nist.gov/)
 
+
```
Comment (sundowndev) [inline README.md:303]: You may want to remove that useless newline

File: CONTRIBUTE.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# Contributing
+# Contribute
 
-Before contributing or comment, please note that I am not a professional infosec or pen tester. I am just a student who want to learn and teach to others. Other contributors might be students as well, so be patient and indulgent with everyone. Do not forget that the main goal here is to write about ethical hacking and teach to others. Tools that are just about malicious hacking and not useful for a professional pen tester will not be added to the README.
+Before contributing or commenting, please note that I am not a professional infosec or pen tester. I am just a student who want to learn and teach to others. Other contributors might be students as well, so be patient and indulgent with everyone. Do not forget that the main goal here is to write about ethical hacking and teach to others. Tools that are just about malicious hacking and not useful for a professional pen tester will not be added to the README.
```
Comment (sundowndev) [inline CONTRIBUTE.md:3]: :+1:

**Comment by sundowndev** [comment]
Thanks for that PR! I left some comments but this seems great, waiting for new changes

File: README.md (hunk 1 above)
Comment (krishna8421) [reply README.md:2]: Should I keep your Old logo or make mine in PNG format.

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (krishna8421) [reply PULL_REQUEST_TEMPLATE.md:5]: It was not Showing as a link to CONTRIBUTE.md

File: README.md (hunk 1 above)
Comment (sundowndev) [reply README.md:2]: Just keep the old one, thx

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (sundowndev) [reply PULL_REQUEST_TEMPLATE.md:5]: Yes but you also changed the filename to CONTRIBUTE.md, what's the matter of that?

Comment (krishna8421) [reply PULL_REQUEST_TEMPLATE.md:5]: Just looks nice

**Comment by krishna8421** [comment]
> Thanks for that PR! I left some comments but this seems great, waiting for new changes

Done, review  it once

**Comment by BigM69** [comment]
Fuck you bitch! Suck my left nut! Sent using the GMX mail appOn 2/17/21 at 2:33 PM, Krishna Kumar wrote:
                
            
            
                
                    From: "Krishna Kumar" <notifications@github.com>Date: February 17, 2021To: "sundowndev/hacker-roadmap" <hacker-roadmap@noreply.github.com>Cc: "Subscribed" <subscribed@noreply.github.com>Subject: Re: [sundowndev/hacker-roadmap] Added Wifite2, SecLists, Image and some minor changes. (#37)
                
                

Thanks for that PR! I left some comments but this seems great, waiting for new changes

Done, review  it once

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/sundowndev/hacker-roadmap/pull/37#issuecomment-780798731",
"url": "https://github.com/sundowndev/hacker-roadmap/pull/37#issuecomment-780798731",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by krishna8421** [comment]
Who the fuck are you asshole, Ask your mother to suck my dick, ohh sorry she already did last night

**Comment by sundowndev** [comment]
Stop this right now. I'll review the PR again soon.

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (sundowndev) [reply PULL_REQUEST_TEMPLATE.md:5]: Please revert the renaming of the `CONTRIBUTING.md` file

File: README.md (hunk 3 above)
Comment (sundowndev) [reply README.md]: Just figured out you were right on this one sorry!

File: README.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
-<p align="center">
-    <img src="https://i.imgur.com/VeEYEkT.png" alt="Hacker roadmap" /><br>
-</p>
+<div style="text-align:center">
+  <img src="https://i.imgur.com/VeEYEkT.png" alt="Hacker Roadmap" />
+</div>
+<br>
```
Comment (sundowndev) [inline README.md:4]: Please revert this unnecessary change, it's not related to this PR
